keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -350,7 +428,28 @@ However, to raise awareness of existing controllers using the seccomp annotation
 migrated, the same warning annotation will be added to the controller as for pods:
 
 ```
-warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.22"
+warning.kubernetes.io/seccomp: "Seccomp set through annotations. Support will be dropped in v1.23"
```

Comment:
This is a bit unusual... I'm not sure we want to annotate like this. There's not a great central place to sweep all workload objects that contain pod templates to get them to mutate this correctly.

* Would we warn if a workload object had both the annotation and the seccomp field set?
* If an update dropped the seccomp annotation, would we clear the warning?
* Would tying this to an [actual warning](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1693-warnings) make more sense once that mechanism is available in 1.19?

Reply:
That is reasonable. I have added a reference to the warning mechanism. 

Given that today is KEP freeze, and this is the only open point, I wonder whether it would be OK to leave as is (with the reference) and I will follow up with a new PR with the details around the warning?

Reply:
Reword this to commit to avoid committing to a specific warning method. You can mention possible mechanisms like a metric, an audit annotation, an annotation on the object, an event, or a warning as described in KEP 1693

Reply:
Added reference to the different mechanisms and removed the previous example.